Update to version 1.2

ps-git opened the pull request:
<!--
Thank you for pull request.
Below are a few things we ask you kindly to self-check before getting a review. Remove checks that are not relevant.
-->
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

Updates release version to latest released (1.2)

ps-git committed:
Update meta.yaml

ps-git committed:
Add Variables

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found it was in an excellent condition.

ps-git committed:
Update meta.yaml

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found it was in an excellent condition.


I do have some suggestions for making it better though...


For **recipe**:

 * Whenever possible python packages should use pip. See https://conda-forge.org/docs/maintainer/adding_pkgs.html#use-pip

ps-git committed:
Update meta.yaml

Update to PIP

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found it was in an excellent condition.

ps-git committed:
Update meta.yaml

ps-git committed:
Update meta.yaml

ps-git committed:
Update meta.yaml

Cleaned up summary

ps-git committed:
Update meta.yaml

Reset package name. 
Update License

CJ-Wright commented on recipe/meta.yaml:
```diff
@@ -1,40 +1,43 @@
 extra:
   recipe-maintainers:
-    - xhochy
+    - ianozsvald
```
why are you removing `xhochy` here?

CJ-Wright commented on recipe/meta.yaml:
```diff
@@ -1,40 +1,43 @@
 
 requirements:
-  host:
+  build:
```
this should be host

ps-git replied:
Incorporate this in latest commit.

CJ-Wright commented on recipe/meta.yaml:
```diff
@@ -1,40 +1,43 @@
 test:
   imports:
-    - ipython_memory_usage
+   - ipython_memory_usage
```
is this correct spacing?

ps-git replied:
This test has been replaced with a direct call to python loading the ipython library..

CJ-Wright commented on recipe/meta.yaml:
```diff
@@ -1,40 +1,43 @@
-  dev_url: https://github.com/ianozsvald/ipython_memory_usage
+  license_file: LICENSE
+  summary:
+    - IPython tool to report memory usage deltas for every command you type. If you are running out of RAM then use this tool to understand what’s happening. It also records the time spent running each command.
```
I think this needs to be a single string

ps-git replied:
This seems ok as is.

xhochy replied:
This is still a list, not a string

CJ-Wright commented:
@conda-forge-admin please rerender

conda-forge-admin committed:
MNT: Re-rendered with conda-build 3.18.10, conda-smithy 3.6.1, and conda-forge-pinning 2019.11.01

ps-git committed:
Update meta.yaml

Updated test. No direct ipython call possible

ps-git committed:
Update meta.yaml

ps-git committed:
Update meta.yaml

Enhanced tests to import ipython_memory_usage module

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found some lint.

Here's what I've got...


For **recipe**:

 * The recipe must have some tests.
 * The test section contained an unexpected subsection name. command is not a valid subsection name.

ps-git committed:
Meta.yaml - Change command to commands

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found it was in an excellent condition.

xhochy commented on recipe/meta.yaml:
```diff
@@ -1,40 +1,45 @@
+  fn: {{ name }}-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 1f4697210257f853fea74de0cf3fae60a32e550e578bac6f46de9b40c550422b
+  sha256: {{ sha256 }}
```
Please keep the sha256 here

ps-git replied:
Updated merge to move this.

xhochy commented on recipe/meta.yaml:
```diff
@@ -1,40 +1,45 @@
   version: {{ version }}
 
 source:
+  fn: {{ name }}-{{ version }}.tar.gz
```
Is this really needed?

ps-git replied:
Updated merge to remove this.

xhochy commented on recipe/meta.yaml:
```diff
@@ -1,40 +1,45 @@
     - ipython
 
 test:
-  imports:
```
Why no import test anymore?

xhochy replied:
From the gitter history it seems this is ok but needs a comment on why.

ps-git replied:
The package is for IPython and in order to succeed tests need to be run with this module.

This has been replaced by importing the library through the python executable.

commands:
    - python -m IPython "import ipython_memory_usage"

ianozsvald replied:
Hi @xhochy as @ps-git notes we need IPython for this to run, I've posted an example on this issue so that in the future I'll issue a useful exception message if it is imported from outside the IPython environment: https://github.com/ianozsvald/ipython_memory_usage/issues/24

ps-git committed:
Clean-up meta.yaml

- Move SHA256 hash into yaml. (As suggested in PR)
- Remove fn: as not required.

xhochy commented:
@ps-git Can you take care of the remaining comments or should I take over?

ps-git commented:
> @ps-git Can you take care of the remaining comments or should I take over?

If you could have a look at the last one. 
  - Replacing 'build:' with 'host:'.

It works fine in conda build as is.

Thank you for your help with this.

xhochy commented:
> > @ps-git Can you take care of the remaining comments or should I take over?
> 
> If you could have a look at the last one.
> 
> * Replacing 'build:' with 'host:'.
> 
> It works fine in conda build as is.
> 
> Thank you for your help with this.

It might work but `host` is meant for "compile-time" dependencies whereas  `build` is for build tools like a compiler.

ps-git commented:
> > > @ps-git Can you take care of the remaining comments or should I take over?
> > 
> > 
> > If you could have a look at the last one.
> > 
> > * Replacing 'build:' with 'host:'.
> > 
> > It works fine in conda build as is.
> > Thank you for your help with this.
> 
> It might work but `host` is meant for "compile-time" dependencies whereas `build` is for build tools like a compiler.

Thank you for clarifying. I will make the change.

ps-git committed:
Update requirements (replace build with host)

xhochy committed:
Make summary a single string
